### PR TITLE
New architecture implemented

### PR DESCRIPTION
**Pull request by jifeon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +36,9 @@
     "debug": "^2.2.0",
     "defaults": "^1.0.2",
     "jsesc": "^0.5.0",
+    "lodash": "^3.10.1",
     "mkdir-p": "0.0.7",
+    "moment": "^2.10.6",
     "phantom": "^0.7.2",
     "phantomjs": "^1.9.18",
     "redparser": "^0.1.0",
```
Comment (maZahaca) [inline package.json]: Is that dependency still required?

File: package.json [hunk 2]
```diff
@@ -37,7 +36,9 @@
     "debug": "^2.2.0",
     "defaults": "^1.0.2",
     "jsesc": "^0.5.0",
```
Comment (maZahaca) [inline package.json]: This one is also to delete

**Comment by maZahaca** [comment]
:boar:

**Comment by maZahaca** [comment]
#2 #3 has been implemented

**Comment by maZahaca** [comment]
#2

**Comment by jifeon** [comment]
You're right, will remove them today

**Pull request merged**